insert_libzstd_ifdefs.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
```

Comment:
Maybe this file should include a header comment describing what it is and why it exists? E.g.=

```
"""
This script rewrites the zstd files ... . When updating zstd, do x y z.
"""
```

Reply:
done